src/hooks/useReportScrollManager/index.js

```diff
@@ -22,13 +22,13 @@ function useReportScrollManager() {
     /**
      * Scroll to the bottom of the flatlist.
      */
-    const scrollToBottom = () => {
+    const scrollToBottom = useCallback(() => {
         if (!flatListRef.current) {
             return;
         }
 
         flatListRef.current.scrollToOffset({animated: false, offset: 0});
-    };
+    }, [flatListRef]);
```

Comment:
As flatlist ref comes from the outside eslint doesn't know it's a ref. Thus it will warn us that we need to add it to the dependency array
Then I'd need to add a eslint-ignore line. When it comes to just adding it which is technically correct, or to suppress a warning I'd choose the former.

Also technically it's not correct. Although right now we don't, the flatlistRef could technically change. In this case we need this function to re-run.